stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -21,11 +21,18 @@ internal class PaymentOptionsViewModel(
     config = args.config,
     prefsRepository = prefsRepository
 ) {
+    /* TODO: Resolve userSelection, newCard, and _selection in SheetViewModel */
     private val _userSelection = MutableLiveData<PaymentSelection>()
     val userSelection: LiveData<PaymentSelection> = _userSelection
 
     override val newCard = args.newCard
 
+    // This is used in the case where the last card ws new and not saved.  In this scenario
```

Comment:
The logic in here could really benefit from some unit testing.  I am working on thinking through the naming a bit more.